### DISCUSSION TITLE
Fire HRR limit in FDS when a fire is growing?

### DISCUSSION THREAD

**Discussion by Ranhill** [firemodels/fds#13277] [Q&A]
Hi,

Is there any HRR limit in FDS when a fire is growing in size? I keep on getting numerical instability error when fire remains around 500 MW in a simulation for around 30s?

 ..............................................................................................
 Time Step:  16800, Simulation Time:     84.22 s
 Time Step:  16900, Simulation Time:     84.69 s
 Time Step:  17000, Simulation Time:     85.12 s
 Time Step:  17100, Simulation Time:     85.58 s
 Time Step:  17200, Simulation Time:     86.08 s
 Time Step:  17300, Simulation Time:     86.55 s

ERROR: Numerical Instability - FDS stopped (CHID: fire_cat)

![image](https://github.com/user-attachments/assets/67379ef1-2464-4c10-920a-a5ca40a67029)

Thanks!

**Comment by drjfloyd** [comment]
There is no hardwired limit for fire size in FDS. However, for a given geometry there may be fire sizes that result in flow fields or other conditions that result in a numerical instability. For example, a fire growing so rapidly in a compartment with a vent opening that compressible flow conditions develop.

**Reply by Ranhill** [reply]
Thank! In the given case, fire is in open condition with domain 100m x 50m x 12 m. Any hint why it stops around 90s even though at 90s the hrr is lower than 10s to 15s before.

**Reply by drjfloyd** [reply]
not much more that can be said without seeing your input file.
